### PR TITLE
Use 1ES pool for pipeline "cosmos-ctl-image-build"

### PR DESCRIPTION
**Pull request by mikeharder**
All pipelines should use 1ES pools for Windows and Ubuntu jobs.

If it's safe to do so, you can run the pipeline on this PR to verify it still runs correctly on the 1ES pool.  I'm not sure what the pipeline does, so I'm not sure what impact running it on this PR might have.

### CODE REVIEW

**Comment by mikeharder** [comment]
Keeping issue active.

**Comment by kushagraThapar** [comment]
@aayush3011 can you please take a look at this issue?
@mikeharder I believe this is blocked on us?

**Comment by mikeharder** [comment]
@kushagraThapar: Yes, this is waiting on the Cosmos team to verify the pipeline still works correctly.  It's just moving the agent used by this pipeline to a different pool (but the same OS image), so I wouldn't expect anything to break, but it should be verified before this is merged.

**Comment by mikeharder** [comment]
You should just need to run the pipeline against this PR, and if everything looks good this can be merged.

**Comment by kushagraThapar** [comment]
Got it @mikeharder , thanks!
@aayush3011  - let's run the pipeline for CTL against this branch and merge this PR, thanks!

**Comment by JonathanGiles** [comment]
@mikeharder What is the status on this PR?

**Comment by mikeharder** [comment]
@JonathanGiles: It would be good to merge this, but it needs to be approved by the Cosmos team.

**Comment by JonathanGiles** [comment]
@kushagraThapar ^^ thanks

**Comment by kushagraThapar** [comment]
@mikeharder / @JonathanGiles apologies for the delay on this, our CTL builds, and pipelines have been broken for quite some time now. I will follow up on this internally and will approve it soon, thanks for your patience!

**Comment by kushagraThapar** [comment]
@mikeharder  - this is the pipeline that needs to be run on this PR - https://dev.azure.com/azure-sdk/internal/_build?definitionId=1813&_a=summary
Can you please schedule a run for this if possible. I don't see an option to run it on this PR.

**Comment by mikeharder** [comment]
> @mikeharder - this is the pipeline that needs to be run on this PR - https://dev.azure.com/azure-sdk/internal/_build?definitionId=1813&_a=summary Can you please schedule a run for this if possible. I don't see an option to run it on this PR.

Run: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2053374&view=results

**Comment by mikeharder** [comment]
@kushagraThapar: The build was successful.  OK to merge?

https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2053374&view=results

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks a lot @mikeharder for running the pipeline and making this change, appreciate it.

**Comment by kushagraThapar** [comment]
> @kushagraThapar: The build was successful. OK to merge?
> 
> https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2053374&view=results

Yes please, approved it. Feel free to merge it or let me know if you need my help on merging it.

**Pull request merged**
